options_test.go

```diff
@@ -63,14 +63,23 @@ var _ = Describe("Options", func() {
 		})
 	})
 
+	Describe("#ChromeOptions", func() {
+		It("should return an Option with ChromeOptions set", func() {
+			config := NewTestConfig()
+			ChromeOptions("args", []string{"v1", "v2"})(config)
```

Comment:
Can you add one more call here, to test [this](https://github.com/sclevine/agouti/pull/108/files#diff-4aae454123f50b906f9aaef44f10011eR38)?